src/stacktrace_libunwind-inl.h

```diff
@@ -52,7 +52,7 @@ _START_GOOGLE_NAMESPACE_
 static bool g_now_entering = false;
 
 // If you change this function, also change GetStackFrames below.
-int GetStackTrace(void** result, int max_depth, int skip_count) {
+size_t GetStackTrace(void** result, size_t max_depth, int skip_count) {
   void *ip;
   int n = 0;
```

Comment:
This change (and the others below) introduces signed/unsigned comparisons, which are A Bad Thing.  For instance, `n` should be changed to be a `size_t`.  There might be other changes needed.  If possible, it would be good to try to compile with a flag that detects these issues.